<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UPE29MM2AppClass</class>
 <widget class="QWidget" name="UPE29MM2AppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1374</width>
    <height>850</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1374</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1374</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UPE29MM2App</string>
  </property>
  <widget class="QLabel" name="backL">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1067</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="upe29mm2app.qrc">:/Resources/Start_form_1.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="migL">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>991</width>
     <height>571</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="upe29mm2app.qrc">:/Resources/2_1224_717.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="nameL">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>1041</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="upe29mm2app.qrc">:/Resources/Name_UPE-29MM2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ButtonLabel" name="synchL">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="upe29mm2app.qrc">:/Resources/Button_Synchronization_original.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ButtonLabel" name="backupL">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="upe29mm2app.qrc">:/Resources/Button_Backup_copy_original.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ButtonLabel" name="restoreL">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="upe29mm2app.qrc">:/Resources/Button_Restore_original.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ButtonLabel" name="startL">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>410</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="upe29mm2app.qrc">:/Resources/Button_Start_original.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ButtonLabel" name="exitL">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>540</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="upe29mm2app.qrc">:/Resources/Button_Exit_original.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="migIconL">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>730</y>
     <width>251</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="upe29mm2app.qrc">:/Resources/MiG.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="sinventL">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>730</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="upe29mm2app.qrc">:/Resources/Sinvent.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ButtonLabel</class>
   <extends>QLabel</extends>
   <header>buttonlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="upe29mm2app.qrc"/>
 </resources>
 <connections/>
</ui>
